### DISCUSSION TITLE
Using GIF with CTK

### DISCUSSION THREAD

**Discussion by hsalnasi** [TomSchimansky/CustomTkinter#1222] [Q&A]
Hello , I was wondering if would it be possible to insert a GIF  using ctk . that would be cool actually

**Comment by Akascape** [comment]
@hsalnasi Try this with customtkinter: https://github.com/olesk75/AnimatedGIF

**Reply by hsalnasi** [reply]
it's working but the quality is pretty bad . also I have already tried implementing it using tkinter and I got the same results 
it would be awesome if the quality is not that laggy

**Reply by Akascape** [reply]
@hsalnasi modify the code by replacing the `tk.PhotoImage` with `PIL.ImageTk.PhotoImage`, and also experiment with other parameters like image interpolation.

**Comment by AliBASE** [comment]
can you please provide some more information how to install AnimatedGIF and how to use it, because I got this error :
NameError: name 'AnimatedGif' is not defined

**Reply by Akascape** [reply]
Place the animatedgif.py file in the directory where your main program is present. Then simply import the class. Or you can copy the source code inside your main program.

**Comment by santhoshu2903** [comment]
by using this, the gif is displaying as image, not running as frames
